docs-markdown/schema/toc.schema.json

```diff
@@ -0,0 +1,97 @@
+{
+	"title": "TOC",
+	"$schema": "https://dotnet.github.io/docfx/schemas/v1.0/schema.json#",
+	"version": "1.0.0",
+	"description": "The schema for a TOC object.",
+	"id": "https://github.com/dotnet/docfx/test/schemas/toc.schema.json",
```

Comment:
It doesn't look like this file exists, but I may not understand how the validator works :wink: